### DISCUSSION TITLE
[Support]: HASS Frigate Proxy Addon - Playback of recordings not possible

### DISCUSSION THREAD

**Discussion by m4nges** [blakeblackshear/frigate#14973] [General Support] [answered]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

According to the HASS log, an alias was removed/changed when updating to HA Core 2025.11 and a deprecated constant was removed when updating to 2025.10. I assume that it is therefore no longer possible to play recordings in Home Assistant itself, as the player is not found / does not work? 

Logger: homeassistant.core
Quelle: helpers/deprecation.py:222
Erstmals aufgetreten: 09:04:49 (1 Vorkommnisse)
Zuletzt protokolliert: 09:04:49

Config was used from frigate, this is a deprecated alias which will be removed in HA Core 2025.11. Use homeassistant.core_config.Config instead, please report it to the author of the 'frigate' custom integration

Logger: homeassistant.components.media_player.const
Quelle: helpers/deprecation.py:222
Integration: Mediaplayer (Dokumentation, Probleme)
Erstmals aufgetreten: 09:04:49 (6 Vorkommnisse)
Zuletzt protokolliert: 09:04:49

MEDIA_CLASS_IMAGE was used from frigate, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaClass.IMAGE instead, please report it to the author of the 'frigate' custom integration

### Steps to reproduce

1.
2.
3.
...

### Version

13.2

### Frigate config file

```yaml
-
```

### docker-compose file or Docker CLI command

```yaml
-
```

### Relevant Frigate log output

```shell
-
```

### Relevant go2rtc log output

```shell
-
```

### Operating system

HassOS

### Install method

HassOS Addon

### Network connection

Wired

### Camera make and model

-

### Screenshots of the Frigate UI's System metrics pages

-

**Comment by hawkeye217** [comment, marked as answer]
You are mixing two unrelated issues here. 

First, the `MEDIA_CLASS_IMAGE` issue has already been fixed for the HA integration: https://github.com/blakeblackshear/frigate-hass-integration/pull/726

Second, the clips being unplayable is a bug in HA that was introduced in 2024.11. The HA folks will need to fix it.

**Reply by m4nges** [reply]
Thanks for pointing me in the right direction! 
I have rolled back to --version=2024.10.3 and everything is working as it should.

**Reply by hawkeye217** [reply]
Great. Happy to help.
